src/init/application/templates/polymer-2.x/test/_element/_element_test.html

```diff
@@ -22,7 +22,7 @@
       suite('<%= elementName %>', function() {
         test('instantiating the element works', function() {
           var element = fixture('basic');
-          assert.equal(element.is, '<%= elementName %>');
+          assert.equal(element.constructor.is, '<%= elementName %>');
```

Comment:
This doesn't actually assert that the element is an `<%= elementName %>` does it?

Just that there is a such an element type and this element is a sub class  of `Polymer.Element`

Also i'm not sure that tests get compiled to ES5 (i've been trying to figure that out too) and the element under test is using ES6 classes